src/main/java/core/basesyntax/service/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import java.util.List;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public List<String> getReport(Map<String, Integer> fruitRepository) {
```

Comment:
move the separator to the constant fields, and add a header. return completed report